node/resolvers/items.ts

```diff
@@ -75,6 +75,7 @@ export const mutations = {
     args: {
       items: OrderFormItemInput[]
       marketingData: Partial<OrderFormMarketingData>
+      salesChannel?: string
```

Comment:
How about refactoring this to use the generated type `MutationAddToCartArgs`?

Reply:
it doesn't work properly, such as with types introduced in the own PR. If I run `vtex setup --typings` the `MutationAddToCartArgs` would have the `salesChannel` correctly, but the toolbelt would add to the package.json a linked version of the package which would break the build

Reply:
true... what if we could merge the PR with the linked version of the package and a bot fixed it (like the release bot?)

Reply:
Definitely not something to be done in this PR of course :^

Reply:
that would be nice, but could potentially break the build for the PR if the bot reverted back to the previous released version... something to discuss :-)